test/e2e/frontend/cypress/tests-ee/team/library.spec.js

```diff
@@ -133,8 +142,6 @@ describe('FlowForge - Library', () => {
         it('allows users to inspect existing Team Libraries ', () => {
             interceptLibraries([], 'folder')
 
-            cy.get('[data-el="ff-tab"]').contains('Team Library').click()
```

Comment:
Because we're now redirected to the shared library page when we access team/library url, we don't need to click on the Team Library navigation tab in order to get to that page, making the rest of the subsequent assertions valid again. The same is valid for the rest of the test in this group.